wouso/interface/cpanel/views.py

```diff
@@ -1409,3 +1412,71 @@ def karma_group_view(request, id):
     return render_to_response('cpanel/karma_group.html',
                               {'form': form, 'group': group},
                               context_instance=RequestContext(request))
+
+class ProposedView(TemplateView):
+
+    template_name = 'cpanel/proposed_home.html'
+
+    def get_context_data(self, **kwargs):
```

Comment:
@vladeugeniu this is not how it should be done. Category & questions fetching should be done with a for. Same applies for displaying them in the html template.